hooks/update_config.js

```diff
@@ -23,16 +23,23 @@ module.exports = function(context) {
         pluginConfigurationFile = path.join(context.opts.plugin.dir, 'plugin.xml'),
         androidPlatformDir = path.join(context.opts.projectRoot,
             'platforms', 'android'),
-        projectConfigurationFile = path.join(context.opts.projectRoot,
-            'config.xml'),
+        projectConfigurationFile,
```

Comment:
Why was this change made? I don't see `projectConfigurationFile` being set anywhere now.